pkg/sql/catalog/lease/lease_test.go

```diff
@@ -1011,11 +1019,6 @@ INSERT INTO t.kv VALUES ('a', 'b');
 	}
 	tableDesc := desctestutils.TestingGetPublicTableDescriptor(kvDB, keys.SystemSQLCodec, "t", "kv")
 
-	{
```

Comment:
removing this because if the the data is gc'ed, you would get `batch timestamp must be after replica gc threshold` errors. And GC is not really needed for this part of the test.